elastic/kibana
# Onboard "SLO burn rate" rule type with FAAD

Towards: https://github.com/elastic/kibana/issues/169867

This PR onboards "SLO burn rate" rule type with FAAD.

## To verify

Create an SLO by using a test index (create a dataview for it), use very low `budget consumed %`
The rule bound to the SLO should create an alert and save it under `.internal.alerts-observability.slo.alerts-default-000001`
